fake_modules/qwik-city/adapters/vercel-edge/vite/index.mjs

```diff
@@ -62,22 +72,39 @@ function vercelEdgeAdapter(opts = {}) {
       }
       const vcConfigPath = join(serverOutDir, ".vc-config.json");
       const vcConfig = {
```

Comment:
This is the vcConfig object which will be written to `.vc-config` file for vercel, based on the `vcEnv` config provided by user in file : `adapter/vercel-edge/vite.config.ts` , (default to edge)